lib/cadet_web/controllers/assessments_controller.ex

```diff
@@ -17,6 +17,18 @@ defmodule CadetWeb.AssessmentsController do
     end
   end
 
+  def unsubmit(conn, %{"submissionid" => submission_id}) when is_ecto_id(submission_id) do
```

Comment:
Not sure if this should be part of assessments or grading. @tuesmiddt?

Reply:
I would argue that it should be part of grading, because anything that takes in submission_id so far are in grading.

Reply:
hmmm i put it in assessment  because it seems to undo `finalize_submission`. but point taken we can put it in grading instead.